cata-blocks.php

```diff
@@ -85,3 +85,33 @@ function cata_blocks_proxy_request( WP_REST_Request $request ) {
 	return new WP_REST_Response( json_decode( $body ), 200 );
 
 }
+
+/**
+ * Add UTMs to Product Block links
+ *
+ * @param string $link Link to be modified.
+ * @return string $new_link Modified product link.
+ */
+function cata_add_product_link_utm( string $link ) {
```

Comment:
Sorry I should have been more specific. I think this would be left on the site that's actually using the blocks. In the plugin, we're just making it available to use.

Reply:
Oh, ok. I see.

So the callback function should be in the child theme, the same as before, but we rename it `cata_add_prod...`  using the cata prefix?

So the [line in render_product() in the shop block that applies the filter](https://github.com/thoughtis/cata-blocks/blob/eb49536c3407d474cb5f73f7cdcc1018e2c0aeff/blocks/shop/shop.php#L89) can call it from the name `cata_product_block_link`?

Reply:
Right, exactly

Reply:
Ok, so this will require a small PR and update to CC (and any other child theme) before this new block is usable then, right?

Reply:
Yes

Reply:
Ok, done. I have the code ready to be submitted as a a PR on the CC from my local testing (once the cata-blocks plugin updated is merged to main)

Reply:
@douglas-johnson I think this block is ready to be merged (and then vers bumped).

Do you concur?